stories/components/NavigationStack/NavigationStack.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import { Example } from './NavigationStack';
+
+export default {
+    title: 'components/NavigationStack',
+    component: Example,
+} as ComponentMeta<typeof Example>;
+
+export const Default: ComponentStory<typeof Example> = args => <Example {...args} />;
+
+Default.args = {};
+
+Default.parameters = {
+    docs: {
+        source: {
+            code: `
+    const { NavigationStack, View, Button, Text } = useMolecules();
+    const ref = useRef<NavigationStackHandle>(null);
+
+    const onPressProfile = () => {
+        ref.current?.navigate('profile');
+    };
+
+    const onGoBack = () => {
+        ref.current?.goBack();
+    };
+
+    return (
+        <NavigationStack ref={ref} initialRouteName="home" {...props}>
```

Comment:
improve the example.
Make it look better.

NO INLINE STYLES

Reply:
also it would be nice to display the imports

Reply:
styles is undefined, make the example work out of the box

Reply:
updated